guides/react_checkout_guide/05_putting_it_togheter.md

```diff
@@ -0,0 +1,161 @@
+### Putting it together
+
+For the client to work with the backend we need to make some adjustment to the code. In your client make sure that you add product_id instead of the id.
+
+```js
+async addToOrder(event) {
+	let id = event.target.parentElement.dataset.id
+  let result = await axios.post('http://localhost:3000/api/orders', { product_id: id } )
+	this.setState({message: {id: id, message: result.data.message}})
+}
+```
```

Comment:
Seems strange to have this code here, perhaps move it to the previous section or give some more intro

Reply:
Remove this section, it gets added later on.